<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_1A">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1B">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_T2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_T1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_T3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2A">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2B">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_T4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1C">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2C">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3A">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3C">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3B">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4A">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4C">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4B">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>16</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(255, 170, 0);border: rgb(255, 170, 0);}  
QPushButton:hover{background-color: rgb(255, 130, 0);border: rgb(255, 135, 0);}  
QPushButton:pressed{background-color: rgb(255, 100, 0);border: rgb(255, 100, 0);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_00">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_01">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_03">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_02">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuchess">
    <property name="title">
     <string>chess</string>
    </property>
    <addaction name="actionstart"/>
   </widget>
   <addaction name="menuchess"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionstart">
   <property name="text">
    <string>start</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
